blobfuse2-release.yaml

```diff
@@ -527,3 +532,45 @@ stages:
                   changeLogType: 'commitBased'
                   isDraft: false
                   assetUploadMode: replace
+  - stage: UpdateLatestVersion
+    dependsOn: ReleaseArtifact
+    condition: succeeded('ReleaseArtifact')
+    jobs:
+      - job: UpdateVersion
+        pool:
+          vmImage: 'ubuntu-18.04'
+        variables:
+          - group: NightlyBlobFuse
+        
+        steps:
+          - checkout: none
+
+          - script: |
+              echo ${{ parameters.tag }}
+            displayName: 'Tag Name'
+
+          - ${{ if eq(parameters.update_version, true) }}:
+              - script: |
+                  sudo apt install python3
+                  python3 --version
+                displayName: 'Installing Python'
+
+              # Adding '-preview' to pre-release versions. For example, 2.0.0-preview.1
+              # Also considering that the tag and version are same
+              - script: |
+                  python3 -c "import json; import requests; import sys;
+                  if(len(sys.argv)!=3):
+                      print('Less arguments')
+                      sys.exit(1)
+                  url = sys.argv[1]
+                  version = sys.argv[2].replace('blobfuse2-', '')
+                  if(len(version)==0):
+                      print('Incorrect Release Tag')
+                      sys.exit(1)
+                  dataDict = requests.get(url).json()
+                  dataDict['blobfuse2'] = version
+                  resp = requests.put(url, data=json.dumps(dataDict, indent=4), headers={'Content-Type': 'application/json', 'x-ms-blob-type': 'BlockBlob'})
```

Comment:
what kind of authorization are we doing here? Is this file not only read only for public access?

Reply:
We are using a SAS url which will have read and write access to the file